test/validation/api/ipsec/ipsec.c

```diff
@@ -987,53 +1007,21 @@ void ipsec_check_out_in_one(const ipsec_test_part *part,
 	odp_packet_t pkto[num_out];
 	int i;
 
-	num_out = ipsec_send_out_one(part, sa, pkto);
+	num_out = ipsec_check_out(part, sa, pkto);
 
 	for (i = 0; i < num_out; i++) {
 		ipsec_test_part part_in = *part;
 		ipsec_test_packet pkt_in;
-		odp_ipsec_packet_result_t result;
-
-		if (ODP_PACKET_INVALID == pkto[i]) {
-			CU_FAIL("ODP_PACKET_INVALID received");
-			continue;
-		}
 
-		if (ODP_EVENT_PACKET_IPSEC !=
```

Comment:
Is this intentional? If yes, (assuming that you are removing an existing check), should we add this in commit description?

Reply:
The removed code was duplicate of that in ipsec_check_out(), which this function now calls. The old code called ipsec_send_out_one() which did not do result processing at all. Does this answer your question?

Reply:
Yeah. Thanks for the explanation.